src/content/en/2020/resource-hints.md

```diff
@@ -170,25 +174,28 @@ With `preload` many different content-types can be preloaded and the[full list](
 
 This is likely related to a large group of sites built as Single Page Apps that need the main bundle as soon as possible to start downloading the rest of their JS dependencies. Subsequent usage comes from font at 8%, style at 5%, image at 1%, and fetch at 1%.
 
+{# TODO(authors): Add or remove figure description. #}
 {{ figure_markup(
   image="mobile-as-attribute-values-by-year.png",
-  caption="Mobile \"as\" attribute values by year.",
+  caption="Mobile `as` attribute values by year.",
   description="TODO",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTYAbLxN40s6mNR1jo0XDe_V4siN8TAsx2mryMp5IQmlJ-9O9eJxYROz7Rw6ozyFP6hlIZHxxh95GqX/pubchart?oid=903180926&format=interactive",
   sheets_gid="1829901599",
   sql_file="as_attribute_by_year.sql"
 ) }}
 
-Compared to the trend in 2019, we've seen rapid growth in font and style usage with the `as` attribute. This is likely related to both developers increasing the priority of critical CSS and also combining `preload` fonts with `display:optional` to [improve](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit) Cumulative Layout Shift ([CLS](https://web.dev/cls/)).
+Compared to the trend in 2019, we've seen rapid growth in font and style usage with the `as` attribute. This is likely related to developers increasing the priority of critical CSS and also combining `preload` fonts with `display:optional` to [improve](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit) Cumulative Layout Shift ([CLS](https://web.dev/cls/)).
 
 Be mindful that omitting the `as` attribute, or having an invalid value will make it harder for the browser to determine the correct priority and in some cases, such as scripts, can even cause the resource to be fetched twice.
 
 ### The `crossorigin` attribute
 
 With `preload` and `preconnect` resources that have CORS enabled, such as fonts, it's important to include the `crossorigin` attribute, in order for the resource to be properly used. If the `crossorigin` attribute is absent, the request will follow the single-origin policy thereby making the use of preload useless.
 
+{# TODO(authors): What is the effect of this? Does this mean that 66% of requests fail? Or that 34% are good, but the 66% isn't notable? The implications of this percentage/callout are unclear. #}
```

Comment:
@Zizzamia

Reply:
Great question! So when a developer use `preload` with or without `crossorigin` the request could be fetch twice.
As we mention at some point in
> Be mindful that fonts preloaded without the `crossorigin` attribute will be fetched [twice](https://web.dev/preload-critical-assets/#how-to-implement-relpreload)!

With 34% we just say the fact that of all `preload` the one that use `crossorigin` attribute is 34% of the cases. Which again can mean people are usinging correctly, or not. 

I guess if we had more time we could have look what was the percentage of preload fonts that forgot to set `crossorigin`.